### PR TITLE
Proposed fix for Issue #243

### LINKED ISSUE #243: best_path_as_path_relax always rendered in configuration
https://github.com/netascode/ansible-dc-vxlan/blob/018c06d1761ba5a0e76d547a2ead4affaed0733b/roles/dtc/common/templates/ndfc_vrf_lite_ebgp.j2#L33C80-L33C103

`best_path_as_path_relax` always rendered in configuration even if not provided in data source

### PR DESCRIPTION
**Pull request by GaetanoCarlucci**
Proposed fix to resolve #243

## Related Issue(s)

Proposed fix for Issue #243

## Related Collection Role

* [ ] cisco.nac_dc_vxlan.validate
* [x] cisco.nac_dc_vxlan.dtc.create
* [ ] cisco.nac_dc_vxlan.dtc.deploy
* [ ] cisco.nac_dc_vxlan.dtc.remove
* [ ] other

## Related Data Model Element

* [ ] vxlan.global
* [ ] vxlan.topology
* [ ] vxlan.underlay
* [ ] vxlan.overlay_services
* [x] vxlan.overlay_extensions
* [ ] vxlan.policy
* [ ] defaults.vxlan
* [ ] other

## Proposed Changes

Proposed fix for Issue #243

## Checklist

* [ ] Latest commit is rebased from develop with merge conflicts resolved
* [ ] New or updates to documentation has been made accordingly
* [ ] Assigned the proper reviewers

### CODE REVIEW

File: roles/dtc/common/templates/ndfc_vrf_lite_ebgp.j2 [hunk 1]
```diff
@@ -30,7 +30,7 @@ router bgp {{ MD_Extended.vxlan.global.bgp_asn }}
   {% endif %}
   {% if switch_item.bgp.best_path_as_path_relax %}
     bestpath as-path multipath-relax
-  {% elif switch_item.bgp.best_path_as_path_relax is not defined and (item.bgp.best_path_as_path_relax | default(defaults.vxlan.overlay_extensions.vrf_lites.bgp.graceful_restart)) %}
+  {% elif switch_item.bgp.best_path_as_path_relax is not defined and (item.bgp.best_path_as_path_relax | default(defaults.vxlan.overlay_extensions.vrf_lites.bgp.best_path_as_path_relax)) %}
```
Comment (mtarking) [inline roles/dtc/common/templates/ndfc_vrf_lite_ebgp.j2:33]: Was this originally a typo as the key is completely changing here?

**Review by mtarking — CHANGES_REQUESTED** [review]

File: roles/dtc/common/templates/ndfc_vrf_lite_ebgp.j2 (hunk 1 above)
Comment (GaetanoCarlucci) [reply roles/dtc/common/templates/ndfc_vrf_lite_ebgp.j2:33]: I assume it was originally a typo. I will let @devegupt confirm this.

Comment (devegupt) [reply roles/dtc/common/templates/ndfc_vrf_lite_ebgp.j2:33]: This was a typo, thanks for catching this.

**Review by juburnet — APPROVED** [review]
Approved

**Review by devegupt — APPROVED** [review]

**Pull request merged**